spring-boot-legacy/src/main/java/org/springframework/boot/legacy/context/web/SpringBootContextLoaderListener.java

```diff
@@ -28,24 +30,30 @@
 import org.springframework.web.context.ContextLoaderListener;
 import org.springframework.web.context.WebApplicationContext;
 import org.springframework.web.context.support.GenericWebApplicationContext;
+import org.springframework.web.context.support.StandardServletEnvironment;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletContext;
 
 /**
  * A {@link ContextLoaderListener} that uses {@link SpringApplication} to initialize an
  * application context. Allows Servlet 2.5 applications (with web.xml) to take advantage
  * of all the initialization extras in Spring Boot even if they don't use an embedded
  * container.
- * 
+ *
+ * @author Daniel Cruver
  * @author Dave Syer
  */
 public class SpringBootContextLoaderListener extends ContextLoaderListener {
 
+	private static final Logger log = LoggerFactory.getLogger(SpringBootContextLoaderListener.class);
```

Comment:
Please use commons-logging for Spring libraries.

Reply:
Will fix.